pause-on-lock.sh

```diff
@@ -50,6 +56,11 @@ dbus-monitor --session "type='signal',interface='com.ubuntu.Upstart0_6'" | \
           RHYTHMBOX_PAUSED=0
         fi
       fi
+      # google play music desktop player
+      if qdbus org.mpris.MediaPlayer2.google-play-music-desktop-player /org/mpris/MediaPlayer2 org.mpris.MediaPlayer2.Player.PlaybackStatus | grep 'Paused' &> /dev/null; then
```

Comment:
If you don't check whether `GPMDP_PAUSED` has been set to 1, it might happen that the player starts playing when unlocking even when it was not playing when the screen got locked. Therefore please add a check for that, so the player only starts again if it was paused by the script.